projects/packages/videopress/src/client/state/resolvers.js

```diff
@@ -36,20 +37,33 @@ const { apiRoot } = window?.jetpackVideoPressInitialState || {};
  *
  * @param {object} video         - Video object.
  * @param {object} resolveSelect - Containing the store’s selectors pre-bound to state
+ * @param {object} dispatch      - Containing the store’s actions pre-bound to state
  * @returns {object}               Tokenized video data object.
  */
-async function populateVideoDataWithToken( video, resolveSelect ) {
+async function populateVideoDataWithToken( video, resolveSelect, dispatch ) {
 	if ( VIDEO_PRIVACY_LEVELS[ video.privacySetting ] !== VIDEO_PRIVACY_LEVEL_PRIVATE ) {
 		return video;
 	}
 
-	const { token } = await resolveSelect.getPlaybackToken( video.guid );
+	let playbackToken = await resolveSelect.getPlaybackToken( video.guid );
+
+	if ( playbackToken ) {
+		// let's set the expire time to 24h
+		const playbackTokenExpireTime = playbackToken.issueTime + 1000 * 60 * 60 * 24;
```

Comment:
How safe is relying on the issueTime prop when treating it as a number type?

Reply:
Added a commit to ensure it's a number before doing any math :)